docs/pbc/all/product-information-management/202410.0/base-shop/feature-overviews/product-bundles-feature-overview.md

```diff
@@ -94,6 +94,13 @@ For example:
 
 If bundled items have quantity&nbsp;<span aria-label="and then">></span> 1 then it will be split as separate items in cart.
 
+{% info_block infoBox %}
+
+When a bundle product is added to the cart with a large quantity (for example, 100-200 items), users may experience a slow-down in the cart operations handling.
```

Comment:
Are your referring to a bundle with a lot of products or cart with a lot of products?